hbase-client/src/main/java/org/apache/hadoop/hbase/client/AbstractRpcBasedConnectionRegistry.java

```diff
@@ -272,6 +272,11 @@ private static RegionLocations transformMetaRegionLocations(GetMetaRegionLocatio
       getClass().getSimpleName() + ".getActiveMaster");
   }
 
+  @Override
+  public String getConnectionString() {
+    return "unimplemented";
```

Comment:
Why override with "unimplemented" if this class is `abstract`?

Reply:
Arg, that's a great point... why did I do this? I did it on the master PR as well. Let me get back to you on this.

Reply:
This is present on the master patch, so let me address it as an addendum that starts with master branch.

Reply:
This is addressed in #4025. @joshelser let me know if you want any of your other comments here addressed. Maybe something else in the SpanBuilders, perhaps?